srml/contracts/src/wasm/prepare.rs

```diff
@@ -503,6 +521,49 @@ mod tests {
 		);
 	}
 
+	mod tables {
+		use super::*;
+
+		// Tests below assumes that maximum table size is configured to a certain number.
+		#[test]
+		fn assume_table_size() {
+			assert_eq!(Schedule::default().max_table_size, 16384);
+		}
+
+		prepare_test!(no_tables,
+			r#"
+			(module
+				(func (export "call"))
+				(func (export "deploy"))
+			)
+			"#,
+			Ok(_)
+		);
+
+		prepare_test!(table_valid_size,
+			r#"
+			(module
+				(table 10000 funcref)
+
+				(func (export "call"))
+				(func (export "deploy"))
+			)
+			"#,
+			Ok(_)
+		);
+
+		prepare_test!(table_too_big,
+			r#"
+			(module
+				(table 20000 funcref)
```

Comment:
Oo, didn't know that you already can use `funcref` instead of `anyfunc`

Reply:
I didn't know that you can use `anyfunc` instead of `funcref`. :-O

Reply:
Hehe, I think `funcref` is a better name though